### ISSUE TITLE
The AUC of CelebDF

### ISSUE THREAD

**Issue by JieLiu95** [mapooon/SelfBlendedImages#7]
I run the inference on Celeb-DF-v2 with your pre trained model, but the AUC is 0.9235, which is lower than yours. And the versions of torch and torchvision are the same with you. Do you know the reason? Can you provide the version of opencv?

**Comment by mapooon** [comment]
The version of opencv (opencv-python) is 4.5.4.60.
It is  strongly recommended to use the provided docker file to reproduce the experimental results.

**Comment by JieLiu95** [comment]
Thanks. This is not caused by the version of opencv, but I have solve it when I change the version of efficientnet-pytorch. Thanks a lot.

**Comment by YU-SHAO-XU** [comment]
@JieLiu95 Can you provide the version of efficientnet-pytorch!?

**Comment by JieLiu95** [comment]
> 

0.7.1

**Comment by YU-SHAO-XU** [comment]
@JieLiu95  and your SBI (train) is 1421 !? i am sure that landmark augmentation is used because show 'exist library' .  I train from scratch and test on CDF AUC only  90.24, so i wanna check if development environment or datst have problem thank you!

**Comment by YU-SHAO-XU** [comment]
@JieLiu95  did u train from scratch ? or only use weight that author provide?!

**Comment by JieLiu95** [comment]
> @JieLiu95 and your SBI (train) is 1421 !? i am sure that landmark augmentation is used because show 'exist library' . I train from scratch and test on CDF AUC only 90.24, so i wanna check if development environment or datst have problem thank you!

I train it from scratch. Did you utilize the mask of X-ray as illustrated in Step 4.
mkdir src/utils/library
git clone https://github.com/AlgoHunt/Face-Xray.git src/utils/library

**Comment by YU-SHAO-XU** [comment]
@JieLiu95  
is that bi_online_generation.py file ? if use that file AUC will  77.xx increase to 90.xx, or you mean DeepFakeMask.py !? and how to use DeepFakeMask !?

**Comment by JieLiu95** [comment]
> If you use bi_online_generation.py, I guess you have used DeepFakeMask.py by the function random_get_hull.

**Comment by YU-SHAO-XU** [comment]
@JieLiu95   so i think it seems to dataset or development environment ~ thank you !

**Comment by JieLiu95** [comment]
> 

Maybe. If you solve this problem, could you share the solution? I'm also very interested in it.

**Comment by YU-SHAO-XU** [comment]
@JieLiu95 
did u hvae error like : 
1. (-215:Assertion failed) !ssize.empty() in function 'resize' 
2. [W pthreadpool-cpp.cc:90] Warning: Leaking Caffe2 thread-pool after fork. (function pthreadpool)
is it normal !?
